synapse/lib/stormhttp.py

```diff
@@ -250,6 +250,10 @@ class LibHttp(s_stormtypes.Lib):
                   'returns': {'type': 'str', 'desc': 'The reason phrase for the status code.', }}},
     )
     _storm_lib_path = ('inet', 'http')
+    _storm_lib_perms = (
+        {'perm': ('inet', 'http', 'proxy'), 'gate': 'cortex',
+         'desc': 'Permits a user to use a proxy with `$lib.inet.http` APIs.'},
```

Comment:
Maybe something like ``Permits a user to specify the proxy used with `$lib.inet.http` APIs`` ?